walkeeper/src/timeline.rs

```diff
@@ -587,12 +587,8 @@ impl Storage for FileStorage {
             let segno = start_pos.segment_number(wal_seg_size);
             // note: we basically don't support changing pg timeline
             let wal_file_name = XLogFileName(PG_TLI, segno, wal_seg_size);
-            let wal_file_path = self.conf.timeline_dir(&ztli).join(wal_file_name.clone());
-            let wal_file_partial_path = self
-                .conf
-                .timeline_dir(&ztli)
-                .join(wal_file_name.clone() + ".partial");
-
+            let wal_file_path = self.timeline_dir.join(wal_file_name.clone());
```

Comment:
these three lines (wal_file_name, wal_file_path, wal_file_partial_path) are repeated three times, probably worth it to extract them into a separate function

Reply:
ok